lib/src/bit_array.dart

```diff
@@ -28,6 +28,29 @@ class BitArray extends BitSet {
     return BitArray._(data);
   }
 
+  /// Creates a bit array using a Uint8List.
+  factory BitArray.fromUint8List(Uint8List list) {
+    final data =
+        list.buffer.asUint32List(list.offsetInBytes, list.lengthInBytes >> 2);
+    return BitArray._(data);
+  }
+
+  /// Create a bit array from a binary string.
+  factory BitArray.fromString(String bitString) {
```

Comment:
Nit: maybe `fromBinaryString` or even better `parseBinary` would be a better name for it - and it should also throw `FormatException` if the format is not right.

Alternatively, `parse`  with a `radix` parameter (similar to `int.parse`'s) could do, and then we could have binary octet or hexadecimal as well.

Reply:
I will change to `parseBinary` for now, and add `radix` in a future PR.